lms/templates/ccx/enrollment.html

```diff
@@ -51,7 +51,7 @@
   <div class="auth-list-container active">
     <div class="member-list-widget">
       <div class="member-list">
-        <h2> ${_("Student List Management")}</h2>
+        <h2 role="heading" tabindex="0"> ${_("Student List Management")}</h2>
```

Comment:
According to Mark's comment on https://github.com/edx/edx-platform/pull/7895#discussion-diff-30075219 you should remove this role attribute.